nextflow.config

```diff
@@ -123,7 +123,7 @@ params {
 
     // Max resource options
     // Defaults only, expecting to be overwritten
-    max_memory                 = '128.GB'
+    max_memory                 = '256.GB'
```

Comment:
Can we revert this change too @pabloaledo ? Shouldn't make a difference setting this higher.